app/assets/javascripts/discourse/app/form-kit/components/fk/control/radio-group.gjs

```diff
@@ -0,0 +1,29 @@
+import Component from "@glimmer/component";
+import { hash } from "@ember/helper";
+import FKText from "discourse/form-kit/components/fk/text";
+import FKControlRadioGroupRadio from "./radio-group/radio";
+
+// eslint-disable-next-line ember/no-empty-glimmer-component-classes
```

Comment:
I can use template only components, as I needed a class name for type checking

Reply:
I removed the comment. eslint is happy now that the class has a static property